src/antsibull/constants.py

```diff
@@ -25,3 +25,27 @@
 ANSIBLE_FORUM_URL = "https://forum.ansible.com"
 
 COLLECTION_EXCLUDE_DIRS = ("docs", "tests")
+
+SANITY_TESTS_DEFAULT: tuple[str, ...] = (
+    "ansible-doc",
+    "compile",
+    "validate-modules",
+    "yamllint",
+)
+SANITY_TESTS_BANNED_IGNORES = frozenset(
+    {
+        "validate-modules!skip",
+        "validate-modules:doc-choices-do-not-match-spec",
+        "validate-modules:doc-default-does-not-match-spec",
+        "validate-modules:doc-missing-type",
+        "validate-modules:doc-required-mismatch",
+        "validate-modules:mutually_exclusive-unknown",
+        "validate-modules:no-log-needed",
+        # Don't enforce this for now. Modules may have private parameters that
+        # are only used by a corresponding action plugin.
+        # "validate-modules:nonexistent-parameter-documented",
```

Comment:
Private parameters are covered by `validate-modules:undocumented-parameter`.
```suggestion
        "validate-modules:nonexistent-parameter-documented",
```